frontend/src/metabase/App.styled.tsx

```diff
@@ -1,41 +1,39 @@
 import styled from "@emotion/styled";
-import { css } from "@emotion/react";
-import Icon from "metabase/components/Icon";
 import { color } from "metabase/lib/colors";
 
-const coreAppCss = css`
-  display: flex;
-`;
-
 export const AppContentContainer = styled.div<{ isAdminApp: boolean }>`
+  display: flex;
   position: relative;
-  height: 100vh;
-
-  ${props => !props.isAdminApp && coreAppCss}
+  height: calc(100vh - 61px);
```

Comment:
Can we extract this `61px` to a constant? So it's something like `calc(100% - ${SOME_ELEMENT_HEIGHT})`